### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LappleApple**
Proofread through the end of the list, continuing previous work. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,17 +78,17 @@ A curated list of awesome Java frameworks, libraries and software.
...
 
-*In existence since the beginning of time and which will continue being used long after the hype has waned.*
+*In existence since the beginning of time, and which people will continue using long after the hype has waned.*
 
 * [Apache Ant](http://ant.apache.org/) - Build process management with XML.
-* [Apache Hadoop](http://hadoop.apache.org/) - Storage and large-scale processing of data-sets on clusters of commodity hardware.
+* [Apache Hadoop](http://hadoop.apache.org/) - Storage and large-scale processing of data sets on clusters of commodity hardware.
 * [Apache OpenNLP](https://opennlp.apache.org/) - Toolkit for common tasks like tokenization.
-* [Apache Velocity](http://velocity.apache.org/) - Templates for HTML pages, emails or source code generation in general.
+* [Apache Velocity](http://velocity.apache.org/) - A Java-based template engine for HTML pages, emails or source code generation.
```
Comment (jbduncan) [inline README.md]: @LappleApple It's implied from it being in awesome-java that Velocity is Java-based, so would you kindly shorten it to "Template engine for...". :)

**Comment by jbduncan** [comment]
Hi @LappleApple, I've just done a quick review. I've not reviewed everything yet, as this is a very large PR, and I'm not entirely sure that I will have the time and energy to fully review it as I'll be busy with other things in the foreseeable future.

@akullpp Is this PR manageable for you, or will @LappleApple need to split it up into multiple PRs or something?

File: README.md (hunk 1 above)
Comment (LappleApple) [inline README.md]: @jbduncan +1 and done :)

**Comment by jbduncan** [comment]
Hi again @LappleApple, I'm impressed by the effort you've put into this, but I've realised that this PR is so large that I'm struggling to properly review it.

Would you kindly split it up into smaller, more manageable chunks and submit those as new PRs. Perhaps one PR per section, e.g. one PR for **Ancients**, another for **Bean Mapping**...?

**Comment by LappleApple** [comment]
@jbduncan Sorry, but I won't be able to do that.

**Comment by jbduncan** [comment]
It's not the PR that's different - it's the reviewer. :wink:

The way I personally function means I sometimes struggle to review large PRs like this one, and having them broken down into smaller, logical sub-PRs makes it easier for me.

#509 was accepted, yes, and I did review it, yes, but I admit that only did a cursory scan over it at the time to check for glaring spelling and grammatical errors - I believed that @akullpp, who was the main reviewer, would have been able to catch more subtle errors that I missed. But it's a little different this time, because since then I became a proper collaborator to the awesome-java project, so I want to take this more seriously and review this PR properly.

I realise that this is different from last time, but despite that, I wonder if you would be able to break down this PR into more manageable chunks, so that I can review your contribution properly. :)

**Comment by LappleApple** [comment]
@jbduncan  Yeah, sorry but again, I won't be able to do that.

**Comment by jbduncan** [comment]
@LappleApple Okay, that's fair enough.

@akullpp Would you have the time to review this at some point in the foreseeable future?

**Comment by LappleApple** [comment]
It would work for me if you guys give this a review, merge the PR, then I can look at the list again to ensure all's well. I used to be a newspaper editor/proofreader, I think you can trust me on this stuff.

**Comment by jbduncan** [comment]
Sorry for taking over a week to get back to this - seems it fell off my radar!

> It would work for me if you guys give this a review, merge the PR, then I can look at the list again to ensure all's well. I used to be a newspaper editor/proofreader, I think you can trust me on this stuff.

Awrigh', you've convinced me. :wink:

Are you happy to have a go at resolving the merge conflicts currently present?

**Comment by LappleApple** [comment]
Thanks, @jbduncan -- done!

**Comment by jbduncan** [comment]
@LappleApple Thanks! Merged.

**Pull request merged**